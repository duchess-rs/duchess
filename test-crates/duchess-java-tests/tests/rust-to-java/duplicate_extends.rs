duchess::java_package! {
    package log;

    public class log.Builder extends java.lang.Object, java.lang.Object {} //~ ERROR: duplicate reference
}

fn main() {}
