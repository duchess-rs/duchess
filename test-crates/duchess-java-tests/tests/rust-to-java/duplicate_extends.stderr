error: error in class `log.Builder`: declared interface `java.lang.Object` not found in the reflected superclasses ()
 --> tests/ui/duplicate_extends.rs:4:5
  |
4 |     public class log.Builder extends java.lang.Object, java.lang.Object {}
  |     ^^^^^^

error: error in class `log.Builder`: duplicate reference in 'extends' to 'java.lang.Object'
 --> tests/ui/duplicate_extends.rs:4:5
  |
4 |     public class log.Builder extends java.lang.Object, java.lang.Object {}
  |     ^^^^^^

error: aborting due to 2 previous errors

